### PR TITLE
Week1 assignment submission

### PR DESCRIPTION
**Pull request by sap-prajapati**

### CODE REVIEW

**Comment by anilrao316** [comment]
please add scrrenshot or recordings of working assignment

**Comment by anilrao316** [comment]
please add assignment 1 : how browser works..

**Comment by anilrao316** [comment]
also add meaningful commit descriptions..

File: Week-1/Code/index.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                        <td>3:34</td>
+                        <td>CATTY CLOUD SYNC</td>
+                    </tr>
+                </table>
+            </div>
+
+                <footer>
+                    <img src="https://vif1g.csb.app/src/assets/cattyboard.jpg">
+                    <div>
+                        <h3>If it Ain't Love<br>
```
Comment (anilrao316) [inline Week-1/Code/index.html:81]: please follow bem naming conventions for classes u can refer : http://getbem.com/naming/

**Comment by anilrao316** [comment]
Add readme file for web fundamentals assignment

**Comment by anilrao316** [comment]
i see a big change which is co- authored by other.. can you please explain why?

**Comment by sap-prajapati** [comment]
> i see a big change which is co- authored by other.. can you please explain why?

yes, i was asking some tips from him but it's all done by myself

**Comment by sap-prajapati** [comment]
> Add readme file for web fundamentals assignment

it is already there in the main branch week1 assignment section

**Comment by sap-prajapati** [comment]
> please add scrrenshot or recordings of working assignment
Screenshot 2022-05-18 at 10 36 40 PM

**Pull request closed without merging**